cicd/build.sh

```diff
@@ -215,6 +215,7 @@ cp "$BASE_DIR/cicd/csm_agent.spec" "$TMPDIR"
     CORE_BUILD_START_TIME=$(date +%s)
     mkdir -p "$DIST/csm/conf/service"
     cp "$CONF/setup.yaml" "$DIST/csm/conf"
+    cp "$CONF/alerts_setup.yaml" "$DIST/csm/conf"
```

Comment:
This will move to section where CLI is build

Reply:
Done